iwyu_output.cc

```diff
@@ -537,6 +558,16 @@ void IwyuFileInfo::AddForwardDeclare(const clang::NamedDecl* fwd_decl,
            << internal::GetQualifiedNameAsString(fwd_decl) << "\n";
 }
 
+void IwyuFileInfo::AddUsingDecl(const clang::UsingDecl* using_decl_decl) {
```

Comment:
I still think this should just be `using_decl`. When I try to pronounce this as "using declaration declaration" I get even more confused :-)